cmd/bc-cli/create/create.go

```diff
@@ -22,6 +22,7 @@ import (
 	"github.com/bestchains/bc-cli/pkg/account"
 	"github.com/bestchains/bc-cli/pkg/common"
 	"github.com/bestchains/bc-cli/pkg/depository"
+	market_repo "github.com/bestchains/bc-cli/pkg/market/repository"
```

Comment:
Good package names are short and clear. They are lower case, with no under_scores or mixedCaps.